xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/lightbox/lightbox.css

```diff
@@ -67,7 +67,7 @@
 #lb {
   left: 0;
   margin-left: 0;
-  width: auto;
+  width: 100%;
```

Comment:
I had to change it to 100%, because if I would've left it auto, the modal was occupying just 50-60% of the screen. I tested the modals from the screenshots from XWIKI-10734 and they're all working.